content/docs/command-reference/plots/index.md

```diff
@@ -12,46 +12,37 @@ usage: dvc plots [-h] [-q | -v] {show,diff,modify} ...
 
 positional arguments:
   COMMAND
-    show         Generate plot from a metrics file.
-    diff         Plot differences in metrics between commits.
-    modify       Modify display properties of data-series plots (has no effect on image-type plots).
+    show         Generate plots from target files or plots definitions from `dvc.yaml` file.
+    diff         Show multiple versions of plot data by plotting them in a single image.
+    modify       Modify display properties of data-series plot outputs (has no effect on image-type plots).
 ```
 
-## Types of metrics
-
-DVC has two concepts for metrics, that represent different results of machine
-learning training or data processing:
-
-1. `dvc metrics` represent **scalar numbers** such as AUC, _true positive rate_,
-   etc.
-2. `dvc plots` can be used to visualize **data series** such as AUC curves, loss
-   functions, confusion matrices, etc.
-
 ## Description
 
-DVC provides a set of commands to visualize certain metrics of machine learning
-experiments as plots. Usual plot examples are AUC curves, loss functions,
-confusion matrices, among others.
+DVC provides a set of commands to visualize data produced by machine learning
+experiments. Usual plot examples are AUC curves, loss functions, confusion
+matrices, among others.
```

Comment:
I agree to present `plots` mainly as a data-centric command instead of another type of "metric" (which was always a little confusing). But one use case is still to use `plots` to visualize "2D" pipeline metrics right? So I think we can elaborate a little in this description:

```suggestion
DVC provides a set of commands to visualize data produced by machine learning
projects. Usual plots include AUC curves, loss functions, or confusion
matrices, for example.

Plots are a great alternative to `dvc metrics` when working with multi-dimensional
performance data. They also help you present and compare [experiments] effectively.
```

☝🏼 `[experiments]` link missing